chart/templates/replicated-supportbundle.yaml

```diff
@@ -49,6 +49,11 @@ stringData:
             name: replicated-custom-app-metrics-report
             includeValue: true
             key: report
+        - secret:
+            namespace: {{ include "replicated.namespace" . }}
+            name: replicated-meta-data
+            includeValue: true
+            key: meta 
```

Comment:
looks like the key needs to be the field that you want to select within the secret data, which would be this one https://github.com/replicatedhq/replicated-sdk/pull/163/files#diff-7703c094e1ac2a3c2107936bc0259de5e1088e6408bf3559ae600d06e889ff40R18

https://troubleshoot.sh/docs/collect/secret/

Reply:
Patched: https://github.com/replicatedhq/replicated-sdk/pull/163/commits/c6d3a412b5333d401cb1e8503440811875715333